apps/src/sites/studio/pages/levels/show.js

```diff
@@ -111,6 +112,9 @@ function initPage() {
           PLATFORMS.BOTH
         );
       }
+      const notificationsEnabled = experiments.isEnabled(
```

Comment:
nit: this value is only looked up so it can be passed down the tree to `RubricFloatingActionButton`. Why not look it up within that component instead?

Reply:
we've had performance problems in the past when experiments are checked within react components that render many times (I think it was a progress bubble). so I try to keep experiment checks outside of react for this reason. it also makes the unit test a little bit simpler. Finally experiment flags in the URL are only set if/when the isEnabled method is called so it is nice to put it in a place that will be called reliably. I wouldn't go so far as to say this is a best practice everyone should follow but it is my slight preference for those reasons.